checkpoint-multi-nic/azuredeploy.parameters.json

```diff
@@ -12,7 +12,7 @@
             "value": "changeme"
```

Comment:
Should now be fixed.

From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 29, 2015 1:56 AM
To: Azure/azure-quickstart-templates
Cc: Tamir Zegman
Subject: Re: [azure-quickstart-templates] Use Static public addresses, allocate additional public addresses to simulate web apps. (#1033)


In checkpoint-multi-nic/azuredeploy.parameters.json<https://github.com/Azure/azure-quickstart-templates/pull/1033#discussion_r48514009>:

> @@ -12,7 +12,7 @@

>              "value": "changeme"

@chkp-zegman<https://github.com/chkp-zegman> Can you look at putting in placeholder parameters that will allow us to test this template with our CI? https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/1033/files#r48514009>.


Email secured by Check Point.